code/modules/martial_arts/combos/bearserk/bear_jaws.dm

```diff
@@ -0,0 +1,30 @@
+/datum/martial_combo/bearserk/bear_jaws
+	name = "Bear Jaws"
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_GRAB)
+	explaination_text = "Bite your opponent like a true lunatic, with even more savagery against knocked down targets."
+
+/datum/martial_combo/bearserk/bear_jaws/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(target.IsWeakened() || IS_HORIZONTAL(target))
+		user.do_attack_animation(target, ATTACK_EFFECT_BITE)
+		target.visible_message("<span class='warning'>[user] leaps onto [target] and bites them, how barbaric!</span>",
```

Comment:
This "how barbaric" is weird, since the messages put into chat are supposed to be explicitly what people see, not their reaction really.